objcgen/extensions.cs

```diff
@@ -117,6 +119,24 @@ public static bool HasCustomAttribute (this MemberInfo self, string @namespace,
 		}
 	}
 
+	public static class ParameterInfoExtensions {
+		public static string ExtendedName (this ParameterInfo p, ParameterInfo[] parameters)
```

Comment:
This should stop being an extension method and just a regular method on namegenerator.